cdap-api/src/main/java/io/cdap/cdap/internal/worker/api/RunnableTaskContext.java

```diff
@@ -43,4 +51,9 @@ public void writeResult(byte[] data) throws IOException {
   public String getParam() {
     return param;
   }
+
+  @Nullable
+  public ClassLoader getParentClassLoader() {
```

Comment:
What is it for? Who is "parent"?

Reply:
This is to pass the classloader user for loading the task. I can rename it to getApplicationClassLoader() if this is confusing.